### PR TITLE
[Merged by Bors] - docs: make `sub_neg` refer to `sub_neg_eq_add`

### PR DESCRIPTION
**Pull request by ericrbg**
I'm not a fan of this naming :/

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Order/Group/Defs.lean [hunk 1]
```diff
@@ -890,7 +890,7 @@ alias ⟨lt_of_sub_pos, sub_pos_of_lt⟩ := sub_pos
 #align lt_of_sub_pos lt_of_sub_pos
 #align sub_pos_of_lt sub_pos_of_lt
 
-@[to_additive (attr := simp) sub_neg]
+@[to_additive (attr := simp) sub_neg "For a - -b = a + b, see `sub_neg_eq_add`."]
```
Comment (eric-wieser) [inline Mathlib/Algebra/Order/Group/Defs.lean:893]:
```suggestion
@[to_additive (attr := simp) sub_neg "For `a - -b = a + b`, see `sub_neg_eq_add`."]
```

**Comment by ocfnash** [comment]
bors merge

**Comment by Ruben-VandeVelde** [comment]
I canceled the build because a commit was pushed manually, so bors would have refused to merge, but that seems to have pushed this PR back to the "awaiting review" stage; apologies.

**Comment by ocfnash** [comment]
bors d=ericbg

**Pull request closed without merging**